### PR TITLE
Update library_example_gapic.yaml to use the latest language_settings convention

### PR DESCRIPTION
**Pull request by ethanbao**
The golden test is still running as part of `build_and_push_docker_image` circleci task.

This presubmit test uses `googleapis/artman:latest` docker image. CircleCI doesn't pull the latest artman image every time it runs, causing misleading result.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -272,29 +272,29 @@ jobs:
...
-      - run:
-          name: Run golden test
-          # Set the environment variable so that the cached shared config will be used during the smoketest.
-          command: export RUNNING_IN_ARTMAN_DOCKER=True && py.test test/golden/artman_golden_test.py --googleapis-dir=/usr/src/artman/test/golden/googleapis
-      - store_artifacts:
-            path: /usr/src/artman/test/golden/actual_library_example.golden
-      - store_artifacts:
-            path: /usr/src/artman/test/golden/actual_library_example_legacy.golden
-    working_directory: /usr/src/artman/
+#  golden-test:
```
Comment (geigerj) [inline .circleci/config.yml]: nit: please add a TODO and explanation for reference after this is submitted.

File: test/golden/library_example.golden [hunk 2]
```diff
@@ -44,26 +44,26 @@
 /java/google-cloud-library_example/.gradle/2.12/taskArtifacts/outputFileStates.bin
 /java/google-cloud-library_example/.gradle/2.12/taskArtifacts/taskArtifacts.bin
 /java/google-cloud-library_example/build.gradle
-/java/google-cloud-library_example/build/classes/main/com/google/cloud/example/library/spi/v1/LibraryServiceClient.class
```
Comment (geigerj) [inline test/golden/library_example.golden]: Did you intend to update the golden file while you disable the test?

**Comment by ethanbao** [comment]
It ended up to be a gapic config problem. The gapic yaml used in gapic is using the old style.

File: test/golden/library_example.golden (hunk 2 above)
Comment (ethanbao) [reply test/golden/library_example.golden]: Yes. Because the same golden test is running in another circleci task.

**Pull request merged**